css/css-text/text-autospace/text-autospace-ideogram-alpha-001.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="help" href="https://drafts.csswg.org/css-text-4/#text-autospace-property">
+<link rel="match" href="text-autospace-ideogram-alpha-001-ref.html">
+<link rel="author" title="Vitor Roriz" href="https://github.com/vitorroriz">
+<style>
+    @font-face {
```

Comment:
nit: `<link rel="stylesheet" herf="/fonts/ahem.css">` is preferred.

Reply:
(same in the -ref.html file fwiw, the linter is complaining about it)

Reply:
Fixed